catalog/async/catalog-async-data/src/main/java/org/codice/ddf/catalog/async/data/ProcessDeleteItemImpl.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.async.data;
+
+import org.codice.ddf.catalog.async.data.api.internal.ProcessDeleteItem;
+
+import ddf.catalog.data.Metacard;
+
+public class ProcessDeleteItemImpl implements ProcessDeleteItem {
+
+    private Metacard metacard;
```

Comment:
Should this `metacard` be `final` because any changes to the `metacard` in a `ProcessRequest<ProcessDeleteItem>` will not be stored?